recipes/ndscope/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "ndscope" %}
+{% set version = "0.7.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://git.ligo.org/cds/ndscope.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Add `noarch: python` (because this package is python-version independent), and `entry_points` so that conda-build can handle the console scripts in a platform-independent manner.

```suggestion
  noarch: python
  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
  entry_points:
    - ndscope = ndscope.__main__:main
```

---

Thanks for the review, @duncanmmacleod.

The setup.py already describes the entry_point.  Why does it need to be repeated here?  Shouldn't it be created already as part of the setuptools build?

---

The `entry_points` are handled specially by conda-build to create a platform-dependent script at _install_ time, so it is best practice to repeat it.

[since this package isn't available for windows it isn't critical here, but as I said, best practice].

---

added